67320.2 <c-25#393> Created logger
67320.2 <c-25#393> appl_initialize was called
67320.2 <c-25#393:9> Logger was updated
67320.2 <c-25#393:9> Started intializing cube lib
67320.2 <c-25#393:9> Finished intializing cube lib with worker, starting to work
67320.2 <c-25#393:9> requestCubes => [7] Buffer size: 0
67320.2 <c-25#393:9> Sent requestCubes signal to root
67320.2 <c-25#393:9> Received 1 cubes from root
67320.2 <c-25#393:9> The main loop continues
67320.2 <c-25#393:9> Started solving a cube
67324 <c-25#393:9> Cube failed
67324 <c-25#393:9> Added failed cube
67324 <c-25#393:9> returnFailedCubes => [7] Buffer size: 5
67324 <c-25#393:9> Sent 1 failed cubes to root
67324 <c-25#393:9> Received receivedFailedCubes signal from root
67324 <c-25#393:9> requestCubes => [7] Buffer size: 0
67324 <c-25#393:9> Sent requestCubes signal to root
67324 <c-25#393:9> Received 1 cubes from root
67324 <c-25#393:9> The main loop continues
67324 <c-25#393:9> Started solving a cube
68300.8 <c-25#393:9> appl_interrupt was called
68300.8 <c-25#393:9> Interrupting worker
68300.8 <c-25#393:9> appl_interrupt was called
68300.8 <c-25#393:9> appl_withdraw was called
68300.8 <c-25#393:9> Started withdraw thread
68300.8 <c-25#393:9> STOP (0.00s since last cb)
68300.8 <c-25#393:9> Solving interrupted
68300.8 <c-25#393:9> The main loop continues
68300.8 <c-25#393:9> Leaving the main loop due to an interruption
68300.8 <c-25#393:9> Finished withdrawing lib
68300.8 <c-25#393:9> Enter destructor of CubeWorker
68300.8 <c-25#393:9> Time waiting for messages: 0.000
68300.8 <c-25#393:9> Enter destructor of CubeWorkerInterface
68300.8 <c-25#393:9> Enter destructor of CubeRoot
68300.8 <c-25#393:9> Finished withdraw thread
68362 <c-25#393:9> Enter destructor
68362 <c-25#393:9> Time working: 980.633
68362 <c-25#393:9> Time suspended: 0.000
68362 <c-25#393:9> Joined cleanup thread
68362 <c-25#393:9> Exit destructor
68362 <c-25#393:9> Destructing logger
